src/main/java/pl/psnc/dei/model/Record.java

```diff
@@ -68,4 +99,41 @@ public Import getAnImport() {
     public void setAnImport(Import anImport) {
         this.anImport = anImport;
     }
+
+	public List<Transcription> getTranscriptions() {
+		return transcriptions;
+	}
+
+	public void setTranscriptions(List<Transcription> transcriptions) {
+		this.transcriptions = transcriptions;
+	}
+
+	public enum RecordState {
+
+		NORMAL(0),
+		E_PENDING(1),
+		T_PENDING(2),
+		U_PENDING(3);
```

Comment:
**E**nrich/**T**ranscribe/**U**pdate
And **N**ormal for record that doesn't need any action to be done.

I will add some comments, so it will not be confusing anymore.